dll/win32/shell32/CShellLink.cpp

```diff
@@ -2602,7 +2602,7 @@ HRESULT CShellLink::DoOpen(LPCMINVOKECOMMANDINFO lpici)
     if (unicode)
     {
         LPCMINVOKECOMMANDINFOEX iciex = (LPCMINVOKECOMMANDINFOEX)lpici;
-        if (iciex->lpParametersW && iciex->lpParametersW[0])
+        if (!StrIsNullOrEmpty(iciex->lpParametersW))
```

Comment:
Question: why not having a "StrIsNotEmpty" function, instead of the negative?

Reply:
To me, StrIsNotEmpty might not check the pointer. StrIsNullOrEmpty is very clear about what we are checking and for LPCMINVOKECOMMANDINFO you need to check both cases.